### PR TITLE
Visiview ome tiff

### PR DESCRIPTION
**Pull request by tibuch**
Adds support for Visiview OME-Tiff acquisitions.

### CODE REVIEW

File: src/faim_hcs/visiview/RegionAcquisition.py [hunk 1]
```diff
@@ -69,3 +75,72 @@ def get_yx_spacing(self) -> tuple[float, float]:
...
+            channel = row["channel"]
+
+            tiles.append(
+                StackedTile(
+                    path=file,
+                    shape=self.tile_shape,
+                    position=TilePosition(
+                        time=time_point,
+                        channel=channel,
+                        z=0,
```
Comment (imagejan) [inline src/faim_hcs/visiview/RegionAcquisition.py:120]: This assumes we always have a full zyx stack present in each tif file, right? I think that holds true for all VisiView acquisitions, we should however also check the various acquisition modalities that VisiView provides (e.g. there used to be a "separate files" option, I don't remember it it's just splitting time points or also z planes...; also, what happens with 2D time lapse data? When using nd/stk format, all time frames end up in a single stk file; we need to see if that's still the case with ome.tif...)
Let's follow up on this by opening separate issues. Sample data for various modalities should be present on our file storage.

File: src/faim_hcs/visiview/ome_companion_utils.py [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+import numpy as np
+from lxml import etree
+
+from faim_hcs.io.ChannelMetadata import ChannelMetadata
+from faim_hcs.utils import rgb_to_hex, wavelength_to_rgb
+
+SCHEMA = "{http://www.openmicroscopy.org/Schemas/OME/2016-06}"
+
+
+def get_z_spacing(metadata):
```
Comment (imagejan) [inline src/faim_hcs/visiview/ome_companion_utils.py:14]: How about adding some minimal docstrings to these public utility functions (also highlighting the fact/limitation that we're parsing the first image only)?

**Review by imagejan — APPROVED** [review]
Looks great, @tibuch!
Let's (squash and) merge this and follow up with some new issues for the points I raised in the comments:

* Support all (2D/3D and/or time) modalities that VisiView provides.
* Add some minimal test dataset for VisiView ome.tif.
* Improve documentation of utility methods.

File: src/faim_hcs/visiview/StackedTile.py [hunk 3]
```diff
@@ -1,12 +1,37 @@
...
+            background_correction_matrix_path=background_correction_matrix_path,
+            illumination_correction_matrix_path=illumination_correction_matrix_path,
+        )
+
     def load_data(self) -> NDArray:
-        data = tifffile.memmap(self.path, mode="r")
+        if self._memmap:
+            data = tifffile.memmap(self.path, mode="r")
+        else:
+            data = tifffile.imread(self.path)
```
[the remark this answers is not in the document]
Comment (tibuch) [reply src/faim_hcs/visiview/StackedTile.py:34]: It depends on how the file was written. Not all tiffs can be read with memmap. 

Memory mapped is lazy, but I am not sure how much of a benefit it brings here.

File: src/faim_hcs/visiview/ome_companion_utils.py (hunk 2 above)
Comment (tibuch) [reply src/faim_hcs/visiview/ome_companion_utils.py:14]: That is a good point. Will add before we merge this.

File: src/faim_hcs/visiview/RegionAcquisition.py (hunk 1 above)
Comment (tibuch) [reply src/faim_hcs/visiview/RegionAcquisition.py:120]: Yes, this is the assumption.

I will add an issue to keep track of this. However, I would only add additional functionality on a need-basis.

**Pull request merged**